helpers/src/rpc.rs

```diff
@@ -16,7 +16,7 @@ impl From<DBError> for Error {
         log::debug!("{}", e);
         match e {
             DBError::NotFound => Error::NotFound,
-            DBError::QueryBuilderError(_) => Error::InvalidData,
+            DBError::QueryBuilderError(_) => Error::InvalidInput,
```

Comment:
I would suggest keeping the other one. I did this because of this: https://doc.rust-lang.org/std/io/enum.ErrorKind.html#variant.InvalidInput

Reply:
Mhh, yeah makes sense. As long as we resolve it in a `400` I am fine with it.